htdocs/mrp/mo_card.php

```diff
@@ -168,7 +168,7 @@
 			$res = $object->add_object_linked('mo', $mo_parent->id);
 		}
 
-		header("Location: ".dol_buildpath('/mrp/mo_card.php?id='.((int) $moline->fk_mo), 1));
+		header("Location: ".dol_buildpath('/mrp/mo_card.php?id='.((int) $tmpmoline->fk_mo), 1));
```

Comment:
$tmpmoline is a variable inside a loop. 
This var should not be used outside the loop.